src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basicPrice = PRICE_PER_DAY * days;
```

Comment:
The variable name 'basicPrice' might be slightly misleading because it implies a fixed price. Consider renaming it to 'totalPrice' or 'rentalCost' to better reflect that it represents the total cost before any discounts are applied.